app/src/main/java/com/x8bit/bitwarden/ui/platform/components/card/BitwardenInfoCalloutCard.kt

```diff
@@ -1,36 +1,31 @@
-package com.x8bit.bitwarden.ui.platform.components.text
+package com.x8bit.bitwarden.ui.platform.components.card
 
 import androidx.compose.foundation.background
 import androidx.compose.foundation.layout.padding
 import androidx.compose.foundation.shape.RoundedCornerShape
 import androidx.compose.material3.Text
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
-import androidx.compose.ui.text.TextStyle
 import androidx.compose.ui.text.style.TextAlign
 import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.dp
 import com.x8bit.bitwarden.ui.platform.theme.BitwardenTheme
 
 /**
- * Represents a Bitwarden-styled policy warning label.
+ * Represents a Bitwarden-styled info callout.
  *
  * @param text The text content for the policy warning.
- * @param textAlign The text alignment to use.
- * @param style The text style to use.
  * @param modifier The [Modifier] to be applied to the label.
  */
 @Composable
-fun BitwardenPolicyWarningText(
+fun BitwardenInfoCalloutCard(
     text: String,
     modifier: Modifier = Modifier,
-    textAlign: TextAlign = TextAlign.Center,
-    style: TextStyle = BitwardenTheme.typography.bodySmall,
 ) {
     Text(
```

Comment:
We may want to wrap this in an actual car in the future but for now this works fine.